__tests__/utils/teamListUtils.js

```diff
@@ -0,0 +1,59 @@
+import { shouldBeScored } from '../../app/utils/teamListUtils'
+
+describe('teamListUtils shouldBeScored', () => {
+    it('should be false when there is exactly one team and we are on the first round', () => {
+        // Arrange
+        const aTeam = {
+            isInPlay: (round) => true
+        }
+        const teamList = [aTeam]
+
+        // Act
+        const result = shouldBeScored(teamList, aTeam, 0)
+
+        // Assert
+        expect(result).toBeFalsy()
+    })
+
+    it('should be false when the target team is in the first position or end of array and its the first round', () => {
+        // Arrange
+        const aTeam = {
+            isInPlay: (round) => true
+        }
+        const teamList = [{}, aTeam]
+
+        // Act
+        const result = shouldBeScored(teamList, aTeam, 0)
+
+        // Assert
+        expect(result).toBeFalsy()
+    })
+
+    it('should be false when the target team is in the first position or end of array and its the second round round', () => {
+        // Arrange
+        const aTeam = {
+            isInPlay: (round) => true
+        }
+        const teamList = [{}, aTeam]
```

Comment:
Same question as above.

Reply:
Updated the test name. See previous reply: [here](https://github.com/jjm3x3/AmazingRaceFantasy/pull/30#discussion_r1461430830)